samples/tasks/scenarios/ovn-sandbox/start_sandbox.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": 2,
+    "title": "Start sandbox",
+    "subtasks": [{
+        "title": "Start sandbox on farm 0",
+        "workloads": [{
+            "name": "OvnSandbox.start_sandbox",
+            "args": {
+                "sandbox_start_args": {}
```

Comment:
sandbox_start_args needn't amount of sandbox, because the amount of sandboxes are decided by 'sandbox' context or its 'farm' and 'tag' arguments.